web/modules/custom/joinup_community_content/config/install/joinup_community_content.permission_scheme.yml

```diff
@@ -146,8 +146,7 @@ view:
 # Create operation permission scheme.
 create:
   'node:pre_moderated':
-    # eLibrary value: Only collection facilitators can create new content.
-    0:
+    only_facilitators:
```

Comment:
This value's schema label, from `joinup_community_content.schema.yml`, still uses the term _eLibrary_.